README.md

```diff
@@ -212,6 +212,9 @@ siteMetadata: {
 const USE_DASH_LINE = true;
 // styling: route line opacity: [0, 1]
 const LINE_OPACITY = 0.4;
+// Configure Google Analytics.
+// If you choose not to utilize Google Analytics, simply leave the field empty. By default, it is disabled.
+const GOOGLE_ANALYTICS_TRACKING_ID = '';
```

Comment:
can we addd a bool named `USE_GOOGLE_ANALYTICS` like others and use this to config use ga or not